gnosis/safe-react
# Epic: Address Book v2

An epic PR to track the progress of the Address Book v2 implementation.

The QA will happen on this branch too.

⚠️ Please DO NOT update this branch from development via GitHub ⚠️

## What to test
-   Create safe – safe name is stored in the address book
-   Load safe - safe name is stored in the address book
-   Add an owner - check name in address book
-   Replace an owner - check name in address book
-   Edit safe name in settings
-   Edit owner name in settings
-   Directly create an address book entry
-   Directly edit an address book entry
-   Check that address book name is shown on transaction details
-   Name stored in address book it's used over known address name from client-gateway
-   Deleting a safe using settings button don't delete the address from the address book
-   Name shown in assets "send" is updated
-   Name shown in assets "receive" is updated
-   Name shown in transaction details for owners is correct
-   Name shown in address book "send" is correct
-   Names shown in settings > spending limits are correct
